app/models/service_provider.rb

```diff
@@ -4,6 +4,7 @@ class ServiceProvider < ActiveRecord::Base
   belongs_to :user_group
 
   enum block_encryption: { 'aes256-cbc' => 1 }
+  enum identity_protocol: { 'OpenID Connect' => 0, 'SAML' => 1 }
```

Comment:
Should we make these more slug-like and then add translations for them to get the pretty-printed names? ex `openid_connect` and `saml` as symbols?